docs/03-github-cloud-runner/06-advanced-topics/10-github/01-release-branches.mdx

```diff
@@ -0,0 +1,15 @@
+# Release Branch Configuration
+
+_GitHub Action examples to pull from the required release branch_
+
+#### Develop branch
+
```

Comment:
There's a need for a section like `## Examples` and then `### Develop branch` headinig 3.